lago_python_client/services/response.py

```diff
@@ -44,3 +51,30 @@ def verify_response(response: Response) -> Optional[Response]:
         return None
 
     return response
+
+
+def prepare_object_response(response_model: Type[BaseModel], data: Mapping[Any, Any]) -> BaseModel:
+    """Return single object response - Pydantic model instance with provided data."""
+    return response_model.parse_obj(data)
+
+
+def prepare_index_response(api_resource: str, response_model: Type[BaseModel], data: Mapping[str, Any]) -> Mapping[str, Any]:
+    """Return index response with meta based on mapping data object."""
+    return {
+        api_resource: [
+            prepare_object_response(response_model=response_model, data=el)
+            for el in data[api_resource]
+        ],
+        'meta': data['meta'],
+    }
+
+
+def prepare_create_response(api_resource: str, response_model: Type[BaseModel], data: Sequence[Mapping[Any, Any]]) -> Mapping[str, Any]:
+    """Return response based on sequence of data objects."""
+    # The only usage - ``WalletTransactionClient.create``
```

Comment:
Maybe the `prepare_create_response` is not the best naming in here since we will probably have few new end-points returning a list of resources for a single POST/PUT actions in the future that will not necessarily be `create` actions.

I would suggest something a bit more generic like `prepare_object_list_response`. What do you think ?